test/benchmarks/Benchmarks.Trace/AspNetCoreBenchmark.cs

```diff
@@ -132,6 +160,12 @@ public string SendRequest()
         {
             return null;
         }
+
+        [Benchmark]
+        public string CallTargetSendRequest()
```

Comment:
Quick question, can we remove the benchmarks in this conditional compilation or no?

Reply:
Sadly no, I've just forgot to add it and the CI was complaining while looking for the test. @kevingosse know more about the underlying reason but the way we execute the benchmark tests is based in compiling for `netcoreapp3.1` targeting the `net472` framework and that mix forces us to declare the same benchmarks for both runtimes.